MetaMask/metamask-extension
# UX Multichain: Added Select an action Modal

This PR includes:
1. Select an Action Modal which will include
- Send
- Swap
- Buy (click out of app indicator)
- Bridge (click out of app indicator)

2. For MMI, send is replaced with Stake button
3. Don't show the button buy, send or swap if they are not supported on a certain network


* Fixes [#1199](https://github.com/MetaMask/MetaMask-planning/issues/1199)
* Fixes [681](https://github.com/MetaMask/MetaMask-planning/issues/681)

## Screenshots/Screencaps

![Screenshot 2023-08-23 at 5 38 25 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/59590927-2fdd-4a1a-963f-3013b2a4eb58)


## Manual Testing Steps


- Go to storybook, check for`SelectAction` Modal
- To see the feature, pass this function on any button onClick `dispatch(showImportTokensModal())` and then click the button to see the modal in Action


## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
